### ISSUE TITLE
`pack stacks` lists stacks and their build/run images

### ISSUE THREAD

**Issue by ssisil** [buildpacks/roadmap#24]
**Scenario : View available stacks**

__WHEN__ I run the command `pack stacks`
__THEN__ I will see all the stacks listed in the config.toml file via the following output 
```
Stack ID                          Build Image                          Run Image
-----------------------------------------------------------------------------------------
<id of stack>  (default)        <path to build image>         <path to run image> 
<id of stack>                   <path to build image>         <path to run image> 

```

**Notes**

- If multiple run images exist, list all images in a comma separated list
- The default stack will have `(default)` listed next to the ID.  If no default is set - io.buildpacks.stacks.bionic will show `(default)` next to it

**Comment by shanks3012** [comment]
Will be released in pack@v0.0.9

**Comment by ekcasey** [comment]
released in `pack` `v0.0.9`
